explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -229,16 +288,13 @@ function AccountsCard({ signature }: Props) {
     }
   }, [signature, details, status, fetchDetails]);
 
-  if (!status?.data?.info) {
+  if (autoRefresh === AutoRefresh.BailedOut || !status?.data?.info) {
     return null;
-  } else if (!details) {
+  } else if (autoRefresh === AutoRefresh.Active) {
```

Comment:
I still think it would be nice to show this error card when auto refresh has been bailed out, thoughts?

Reply:
I agree! I added that case. If it doesn't make sense to have a retry button I can pull that out.

Reply:
Retry makes sense! But it should be `refreshStatus` because otherwise the details could get refreshed but the status would be still stuck showing 0 confirmations

Reply:
Oh yes, done!